schemes/chemistry/prescribed_ozone.meta

```diff
@@ -0,0 +1,162 @@
+[ filename ]
+  standard_name = filename_of_prescribed_ozone
+  units = none
+  type = character | kind = len=*
+  dimensions = ()
+  intent = in
+[ filelist ]
+  standard_name = filename_of_file_list_for_prescribed_ozone
+  units = none
+  type = character | kind = len=*
+  dimensions = ()
+  intent = in
+[ datapath ]
+  standard_name = datapath_for_prescribed_ozone
+  units = none
+  type = character | kind = len=*
+  dimensions = ()
+  intent = in
+[ data_type ]
+  standard_name = time_interpolation_method_for_prescribed_ozone
+  units = none
+  type = character | kind = len=*
+  dimensions = ()
+  intent = in
+[ cycle_yr ]
+  standard_name = cycle_year_for_prescribed_ozone
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = in
+[ fixed_ymd ]
+  standard_name = fixed_date_for_prescribed_ozone
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = in
+[ fixed_tod ]
+  standard_name = fixed_time_of_day_for_prescribed_ozone
+  units = s
+  type = integer
+  dimensions = ()
+  intent = in
+[ errmsg ]
+  standard_name = ccpp_error_message
+  units = none
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+[ errflg ]
+  standard_name = ccpp_error_code
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = out
+
+[ccpp-arg-table]
+  name  = prescribed_ozone_run
+  type  = scheme
+[ ncol ]
+  standard_name = horizontal_loop_extent
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ pver ]
+  standard_name = vertical_layer_dimension
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ mwdry ]
+  standard_name = molecular_weight_of_dry_air
+  units = g mol-1
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ boltz ]
+  standard_name = boltzmann_constant
+  units = J K-1
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ t ]
+  standard_name = air_temperature
+  units = K
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ pmiddry ]
+  standard_name = air_pressure_of_dry_air
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ pmid ]
+  standard_name = air_pressure
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ pint ]
+  standard_name = air_pressure_at_interface
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_interface_dimension)
+  intent = in
+[ phis ]
+  standard_name = surface_geopotential
+  units = m2 s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ zi ]
+  standard_name = geopotential_height_wrt_surface_at_interface
+  units = m
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_interface_dimension)
+  intent = in
+[ prescribed_ozone ]
+  standard_name = O3
```

Comment:
Just adding a reminder that we'll probably need to use the constituents object here instead of outputting a specific `O3` variable.

Reply:
Thanks, updated